eaw-types-emmyluadoc.lua

```diff
@@ -378,6 +378,10 @@ function GameObject.Get_All_Projectile_Types() end
 ---@return boolean
 function GameObject.Is_Selectable() end
 ---@public
+--- Sets a unit to be selectable or not selectable by the player
+---@param selectable boolean
+function GameObject.Set_Selectable(selectable) end
```

Comment:
Just curious, does this call crash if you try to set a non-selectable object (so one without the `SELECTABLE` behaviour) selectable?

Reply:
I don't know, never tried it.

Reply:
If it does, we'll probably have to add a warning to the doc.